Makefile

```diff
@@ -288,6 +288,17 @@ static-check: gocyclo
 
 get-deps: .get-deps-stamp
 
+DEP_VERSION=v0.4.1
+get-dep-linux:
+	mkdir -p ./bin
+	curl -L https://github.com/golang/dep/releases/download/$(DEP_VERSION)/dep-linux-386 -o ./bin/dep
+	chmod +x ./bin/dep
```

Comment:
Only downloads the execute file, but it's not added to the system PATH, can we take one more step further and run dep ensure here?

Reply:
> Only downloads the execute file, but it's not added to the system PATH,

why add it to the system path? my intention is for us to use this from within the repo and avoid any conflict with the existing version of dep that might be on the system. please let me know if im missing something though!

> can we take one more step further and run dep ensure here

i'm also not sure about this since `dep ensure` will try to install the project's dependencies and may change the state of the vendor directory. i feel this may create room for confusion. but again, let me know what you think.